ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -84,6 +84,13 @@ def pytest_addoption(parser):
         dest='email',
         help="Email ID to send results",
     )
+    parser.addoption(
+        '--squad-analysis',
```

Comment:
@dahorak please add some info about this parameter also to documentation: https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md

Reply:
Added in 21b7e01.